examples/plot_pole_ladder_S2.py

```diff
@@ -0,0 +1,56 @@
+"""Plot the pole ladder scheme for parallel transport on S2.
+
+Sample a point on S2 and two tangent vectors to transport one along the
+other.
+
+"""
+
+import matplotlib.pyplot as plt
+
+import geomstats.backend as gs
+import geomstats.visualization as visualization
+from geomstats.geometry.hypersphere import Hypersphere
+
```

Comment:
Nice! Could you use the `def main` syntax for the example:
```
if __name__ == "__main__":
    main()
```

to be consistent with other examples?